src/app/marketplace/marketplace-data-product-list/marketplace-data-product-list.component.html

```diff
@@ -1,96 +1,72 @@
 <div *ngIf="dataSource">
 
-  <mat-form-field *ngIf="showSearch">
-    <mat-icon class="search-icon" matPrefix>search</mat-icon>
-    <input matInput (keyup)="applyFilter($event.target.value)" placeholder="Search">
-  </mat-form-field>
+  <div class="list-search" *ngIf="showSearch">
+    <input class="list-search__input" (keyup)="onTypeAhead($event.target.value)" placeholder="Search">
+    <mat-icon class="list-search__icon">search</mat-icon>
+  </div>
 
   <div class="data-container">
     <div class="loading-shade" *ngIf="isLoadingResults">
       <mat-spinner *ngIf="isLoadingResults"></mat-spinner>
     </div>
 
-    <mat-table #table [dataSource]="dataSource" matSort (matSortChange)="sortChanged($event)">
-
-      <!-- Name Column -->
-      <ng-container matColumnDef="name">
-        <mat-header-cell *matHeaderCellDef mat-sort-header [disabled]="disableSorting"> File name</mat-header-cell>
-        <mat-cell *matCellDef="let element"> {{element.name}}</mat-cell>
-      </ng-container>
-
-      <!-- Title Column -->
-      <ng-container matColumnDef="title">
-        <mat-header-cell *matHeaderCellDef mat-sort-header [disabled]="disableSorting"> Title</mat-header-cell>
-        <mat-cell *matCellDef="let element"> {{element.title}}</mat-cell>
-      </ng-container>
-
-      <!-- Categories Column -->
-      <ng-container matColumnDef="category">
-        <mat-header-cell *matHeaderCellDef> Categories</mat-header-cell>
-        <mat-cell *matCellDef="let element"> {{element.category | array_join: ', '}}</mat-cell>
-      </ng-container>
-      <!-- Size Column -->
-      <ng-container matColumnDef="size">
-        <mat-header-cell *matHeaderCellDef mat-sort-header [disabled]="disableSorting"> Size</mat-header-cell>
-        <mat-cell *matCellDef="let element"> {{ element.size | fileSize }}</mat-cell>
+    <table mat-table #table [dataSource]="dataSource" matSort (matSortChange)="sortChanged($event)">
+
+      <!-- Details Column -->
+      <ng-container matColumnDef="details">
```

Comment:
I think sorting won't work on column 'details'. This name should exists in elasticsearch as an index.